app/main.py

```diff
@@ -1 +1,17 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command, file_before, direction = command.split()
```

Comment:
![image](https://user-images.githubusercontent.com/103201010/220118279-b1290967-5a54-422b-a108-e3571d53bf50.png)
u should check if command is correct